<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaytaSaldo</class>
 <widget class="QDialog" name="NaytaSaldo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textAsiakas">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textTapahtumat">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textSaldo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>49</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tilitapahtumat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>67</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Asiakas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
